codesy/base/models.py

```diff
@@ -16,6 +16,7 @@
 
 
 class User(AbstractUser):
+    stripe_cc_token = models.CharField(max_length=100, blank=True)
```

Comment:
This feels messy to have the `stripe_cc_token` field, since we're not ever storing the cc token. But to drop it we either need to customize the DRF serializer for `User`, or add another custom view just for the cc token->account token.

Can we add a `TODO:` here and also make it `default=''` and `editable=False` to help keep us from storing anything in this field in the future?